src/components/Clients/ClientListTableRow.jsx

```diff
@@ -46,13 +49,7 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
 
   // Event handler for deleting client from client list
   const handleDeleteClient = async () => {
-    if (
-      !window.confirm(
-        `You're about to delete ${client.person} from your client list, do you wish to continue?`
-      )
-    ) {
-      return;
-    }
+    setShowClientDeleteModal(false);
     runNotification(`Deleting "${client.person}" from client list...`, 3, state, dispatch);
```

Comment:
Non-Issue: Any thoughts on moving these notifications into the modal? Seems like if we're going to be displaying messages about deletion, we can display them all in one place. Doesn't have to be in this PR if we want to change it.

Reply:
See comment above from Ka Hung and my quickly researched response.